source/appModules/devenv.py

```diff
@@ -89,9 +92,14 @@
 
 
 class AppModule(appModuleHandler.AppModule):
+
+	def _get_major(self):
+		return int(self.productVersion.split(".", 2)[0])

```

Comment:
Rather than having separate attributes for major and minor, could you please make this an attribute that returns a tuple? This can be similar to the code you removed from chooseNVDAObjectOverlayClasses

Reply:
Done, I will upload a change now.